com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAPK.java

```diff
@@ -357,6 +357,8 @@ public static void create(Project project, File bundleDir, ICanceled canceled) t
         ap3.delete();
         ap4.delete();
         FileUtils.deleteDirectory(tmpResourceDir);
+        FileUtils.deleteDirectory(new File(appDir, "libs"));
+        FileUtils.deleteDirectory(resDir);
```

Comment:
We also remove intermediate artefacts folders `res` and `libs` from our bundle folder for apk build